package.json

```diff
@@ -32,7 +32,7 @@
   "devDependencies": {
     "@microsoft/api-documenter": "^7.23.9",
     "@microsoft/api-extractor": "^7.38.0",
-    "@types/jest": "^28.1.1",
+    "@types/jest": "^29.5.11",
```

Comment:
Had to upgrade `jest` and related packages to use its time manipulation capabilities